### PR TITLE
PCHR-2525: Fix UI-Select Multiple Styles

### PR DESCRIPTION
**Pull request by deb1990**
As part of this PR, bugs related to UI-Select Multiple has been fixed.

The dropdown icon has been changed to Search icon, as suggested by Jon.
manager_leave___hr17-4_

### CODE REVIEW

**Comment by AkA84** [comment]
@deb1990 why do we have the problem with the extra line with the new icon but not with the old one?

File: scss/bootstrap/overrides/style/_ui-select.scss [hunk 1]
```diff
@@ -137,12 +141,30 @@
...
+    display: inline-block;
+    max-width: 90%;
+  }
+  
   .ui-select-search {
     padding-left: 6px !important;
   }
 
   &:after {
-    @include dropdown-arrow();
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss]: are we using this mixing anymore?

File: scss/bootstrap/overrides/style/_ui-select.scss [hunk 2]
```diff
@@ -137,12 +141,30 @@
...
+    content: '\f002';
+    bottom: 0;
+    font-family: "FontAwesome";
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss]: i believe fontawesome already provides a mixing that does this

Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss]: also, not sure if we should consider putting this in a mixing like the old `dropdown-arrow`

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: scss/bootstrap/overrides/style/_ui-select.scss (hunk 1 above)
Comment (deb1990) [reply scss/bootstrap/overrides/style/_ui-select.scss]: No, removed.

File: scss/bootstrap/overrides/style/_ui-select.scss (hunk 2 above)
Comment (deb1990) [reply scss/bootstrap/overrides/style/_ui-select.scss]: Done

**Comment by deb1990** [comment]
> @deb1990 why do we have the problem with the extra line with the new icon but not with the old one?

Previously the input was limited to 100px using css, that is why it was not reaching upto the drop down arrow, now as I removed the same, the width set by ui-select(inline width, using JS) is applied, but we also included the arrow, so there is not enough space for the input element, hence it is coming to the next line.
Please let me know if it is still not clear, we can have a call then.

**Comment by igorpavlov-zz** [comment]
@deb1990 the extra line is what I fixed last time. I also made this field multiline and reduced the width of the input to a reasonable amount. But as far as I can see it produced the problem with an unclickable area. The ideal solution would be making the input field (the real input) filling the whole rest width and have a min-width as well. This would require JS though.

Or another idea: clicking on the empty area of the whole block (but not actual tags) should "focus" the input field.

**Comment by deb1990** [comment]
@igorpavlov Thanks for the tip, I fixed it using JS, now I focus on the input whenever the element is clicked. Refer https://github.com/civicrm/civihr/pull/2265

**Review by AkA84 — APPROVED** [review]

**Pull request merged**